soccer/gameplay/tactics/positions/goalie.py

```diff
@@ -106,7 +110,9 @@ def robot_is_facing_our_goal(self, robot):
     # note that execute_running() gets called BEFORE any of the execute_SUBSTATE methods gets called
     def execute_running(self):
         if self.robot != None:
-            self.robot.face(main.ball().pos)
+            #face the ball in all states but intercept
+            if not self.isInter:
```

Comment:
So I've never actually used this, but there might be a function `self.is_in_state(Goalie.intercept)` that returns whether you are in that state or not.

If that doesn't work, you can also just check if `self.has_subbehavior_with_name('intercept')` is true and it should do the same thing.